client/src/app/pages/controls/stakeholders/components/stakeholder-form.tsx

```diff
@@ -132,7 +169,7 @@ export const StakeholderForm: React.FC<StakeholderFormProps> = ({
         };
       });
 
-    const payload: Stakeholder = {
+    const payload: Omit<Stakeholder, "id"> = {
```

Comment:
Seems like this was a good thought, but no reason we should have to remember to omit an ID every place we define an instance type for submitting payloads or setting the types in the rest.ts file. Seems like we could think of something better here.

Reply:
What is wrong with leaving the id optional and conditionally merging it to the payload object when we are updating?

Reply:
There are no need to worry about Ids on any object with this approach.
The reason is because resources always have an ID but at creation time.
Instead until now we've been looking at it from the other end where Ids are optional because we don't know the value at creation time.  
The benefit mean certainty when handling such objects and not having to test if the object is not "undefined".

Reply:
Makes sense - thanks.